script/test

```diff
@@ -7,7 +7,8 @@ ROVER="/usr/local/openresty/luajit/bin/rover"
 function run_busted_tests {
     sudo cp -R /tmp/lua_modules .
 
-    cd "${PROJECT_PATH}" || exit; ${ROVER} exec ./bin/busted
+    cd "${PROJECT_PATH}" || exit; \
+        TEST_NGINX_REDIS_HOST=redis ${ROVER} exec ./bin/busted
```

Comment:
Shouldn't this env var be pushed from outside? Docker compose/makefile ?

Reply:
Yes, what I meant is that if we make the change you propose, everyone should be able to use this script because `redis` will not be hardcoded and will be set depending on the user environment.

Reply:
I see. Sorry.

Reply:
I've moved the envs to the docker-compose file.

Reply:
Yes. I removed the rest and forgot about this one. Fixed.